meteor/client/ui/Settings/ShowStyle/VariantSettings.tsx

```diff
@@ -22,9 +26,12 @@ interface IShowStyleVariantsProps {
 	layerMappings?: { [key: string]: MappingsExt }
 	sourceLayers?: Array<{ name: string; value: string; type: SourceLayerType }>
 }
+
 interface IShowStyleVariantsSettingsState {
 	editedMappings: ProtectedString<any>[]
+	uploadFileKey: number
```

Comment:
There is a mismatch between the name and the type. The name is written as a verb. Verbs indicate that stuff happens when you use them i.e. verbs indicate that is a function or method. This is neither but instead a number. (Kinda the same princible that applies for prefixing booleans with `is` or `has`)

Looking at how it's used it appears to be a timestamp. What kind of timestamp I am not sure since it's used inside `UploadButton` that I have yet to check out.

But it would be nice that the name indicated the fact that it is a timestamp and preferably what it's a timestamp off and that the name isn't a verb when it's not describing a method.

Reply:
Added it to [f74f966](https://github.com/tv2/tv-automation-server-core/pull/188/commits/f74f9665479993af5c8647b78f40a11fe9370e0f).